server/utils.go

```diff
@@ -74,9 +72,9 @@ func (p *Plugin) IsBotDMChannel(channel *model.Channel) bool {
 	return true
 }
 
-func (p *Plugin) sleepUpTo(maxDelay time.Duration) {
-	r := rand.New(rand.NewSource(p.now().UnixNano()))
-	delay := time.Duration(r.Int63n(int64(maxDelay) + 1))
+// func (p *Plugin) sleepUpTo(maxDelay time.Duration) {
```

Comment:
If you could remove this, that'd be great. This was left over from some extremely unreliable code that never made it into production.

Reply:
i will remove that